content/docs/command-reference/exp/run.md

```diff
@@ -134,6 +141,17 @@ CPU cores).
 - `-j <number>`, `--jobs <number>` - run this `number` of queued experiments in
   parallel. Only applicable when used in conjunction with `--run-all`.
 
+- `-r <commit>`, `--rev <commit>` - continue an experiment from a specific
+  checkpoint name or hash (`commit`). Required for explicitly resuming
+  checkpoints in queued or temp dir runs.
+
+- `--reset` - restart a checkpoint experiment from scratch (resets existing
+  checkpoints to the initial state).
```

Comment:
> resets existing checkpoints

By this do we basically mean delete all checkpoints? A bit confused by the term "reset".

Reply:
The current CLI message for this is
```
--reset               Reset existing checkpoints and restart the experiment.
```

If you have previously generated a series of checkpoints (ex. epochs 1, 2, and 3), normally `dvc exp run` will resume from the repo state at the end of `3` (and your next checkpoint would be epoch 4).

Using `--reset` means the repo will be reset to the initial state prior to checkpoint `1` (so your next checkpoint would be epoch 1), and the previous set of checkpoints will cleared/overridden with this new run.

Reply:
> the repo will be reset
> the previous set of checkpoints will cleared

Yeah thanks, that's what I was guessing. I was confused by the phase "resets existing experiments". The workspace and experiment are reset. The checkpoints are deleted 👍